src/vivarium_csu_alzheimers/components/testing.py

```diff
@@ -179,3 +197,26 @@ def _get_bbbm_testing_rate(self, event_time: pd.Timestamp) -> float:
         rates = [rate for _, rate in BBBM_TESTING_RATES]
 
         return np.interp(event_time.value, timestamps, rates)
+
+    def _generate_bbbm_testing_history(
+        self, simulants: pd.DataFrame, eligible_sims: pd.Series, time_of_event: Time
+    ) -> pd.Series:
+        """Generates previous BBBM test data for new simulants between 0 and 2.5 years prior
+        to entering the simulation. Note that we immediately check if simulants get tested on
```

Comment:
I don't understand the second sentence here.

Reply:
It was outdated.